qemu.mk

```diff
@@ -195,7 +195,7 @@ endif
 check: $(CHECK_DEPS)
 	ln -sf $(ROOT)/out-br/images/rootfs.cpio.gz $(BINARIES_PATH)/
 	cd $(BINARIES_PATH) && \
-		export QEMU=$(ROOT)/qemu/arm-softmmu/qemu-system-arm && \
+		export QEMU=$(ROOT)/build/qemu/arm-softmmu/qemu-system-arm && \
```

Comment:
`$(QEMU_PATH)` not `$(ROOT)`

Reply:
Fixed, thanks!

Reply:
And you forgot to update `$(QEMU_PATH)/config-host.mak:`.
I suggest you define `QEMU_BUILD ?= $(QEMU_PATH)/build` just after `QEMU_PATH` then use that where needed.

Reply:
Thanks for the advice! I've made the changes.